nurseconnect/templates/registration/login.html

```diff
@@ -4,27 +4,32 @@
 
 
 {% block content %}
+    <div class="Article-body">
+        <h1 class="Article-title">Welcome back. Please enter your login details.</h1>
+    </div>
     <form method="post" action=".">{% csrf_token %}
         <div class="FormGroup FormGroup--padded">
             {% if form.errors %}
                 <p class="error errorlist">{% trans "Your mobile number and password does not match. Please try again." %}</p>
             {% endif %}
             <div class="FormField FormField--center FormField--bordered">
-                <p><strong>{% trans "Username" %}</strong></p>
```

Comment:
You can use the label's you've specified in the forms.
`<p><strong>{% trans "{{ form.username.label_tag}}" %}</strong></p>`